### PR TITLE
Adding ontology for audit event types

### PR DESCRIPTION
**Pull request by escowles**
Addresses https://jira.duraspace.org/browse/FCREPO-1422

### CODE REVIEW

File: audit.rdf [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    <rdfs:label xml:lang="en">content modification</rdfs:label>
+    <skos:prefLabel xml:lang="en">content modification</skos:prefLabel>
+    <rdfs:comment xml:lang="en">A content file is modified.</rdfs:comment>
+    <skos:definition xml:lang="en">A content file is modified.</skos:definition>
+    <skos:inScheme rdf:resource="http://id.loc.gov/vocabulary/preservation/eventType"/>
+    <skos:inScheme rdf:resource="http://id.loc.gov/vocabulary/preservation"/>
+    <rdf:type rdf:resource="http://www.w3.org/2004/02/skos/core#Concept"/>
+  </owl:NamedIndividual>
+
+  <owl:NamedIndividual rdf:about="http://fedora.info/definitions/v4/audit#derivativeCreation">
```
Comment (awoods) [inline audit.rdf]: Do you expect #derivativeCreation and #indexing to be #ExternalEvents?

Comment (escowles) [inline audit.rdf]: Yes, that's the pattern I would normally expect -- though there may be some people using sequencers to effectively make these internal events.

**Comment by awoods** [comment]
@ajs6f, in line with the "API partitioning" effort, we will clearly be moving this ontology over to something like "fcrepo4-extras"; but in the meantime, do you or @acoburn have any comments on the content of this document?

**Comment by ajs6f** [comment]
I have little to contribute. I do find it somewhat bizarre that "derivativeCreation" and "indexing" are considered events for which one would audit a _repository_. I assume that there's some good reason for this, and we're not just conflating the auditing of a repository with the auditing of the larger system in which it is a component.

**Comment by acoburn** [comment]
I would echo @ajs6f. I don't have anything against these events, but they seem to be related to a different layer in a DAMS, distinct from the underlying repository itself.

**Comment by awoods** [comment]
Yes, there seem to be two scopes at play:
- the repository
- the resource
The Audit Service is establishing a pattern for associating events to resources within the repository as well as at-large. Maybe we should separate the "external events" from this current effort... while still providing recipes for how integrators can push external events into the associated triplestore.

**Comment by escowles** [comment]
I'm fine with removing indexing and derivativeGeneration from the ontology -- and saying that implementers can define their own local types for whatever events they want to track.

**Comment by ajs6f** [comment]
The beauty of using Web ontology languages for this purpose is precisely that people can extend them however they want, with or without our input.

**Comment by awoods** [comment]
Resolved with: https://github.com/fcrepo4/ontology/commit/53756a205e007d4d9f2dca55f9df0cab8f2ef69f

**Pull request closed without merging**